community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/command/IndexTransactionApplier.java

```diff
@@ -103,15 +106,42 @@ private void updateIndexes() throws IOException, IndexCapacityExceededException,
 
     private void updateLabelScanStore() throws IOException, IndexCapacityExceededException
```

Comment:
You could remove these exception declarations, since they are no longer thrown from this method.